documentation/asciidoc/topics/proc_adding_backup_locations.adoc

```diff
@@ -0,0 +1,80 @@
+[id='adding_backup_locations-{context}']
+= Adding Backup Locations to Caches
+When you configure {brandname} clusters to perform cross-site replication, you
+can add backup locations to your cache configurations.
+
+Backup locations can automatically go offline when remote sites become unavailable. This prevents nodes from attempting to replicate data to offline backup locations, which can have a performance impact on your cluster because it
+results in error.
+
+You configure how long to wait before backup locations go offline. A good rule
+of thumb is one or two minutes. However, you should test different wait periods
+and evaluate their performance impacts to determine the correct value for your
+deployment.
+
+For instance when {openshiftshort} terminates a site master node, that backup
+location becomes unavailable for a short period of time until {ispn_operator}
+elects a new site master. In this case, if you configure too small an interval
```

Comment:
can you reivew the sentence started "In this case,"? I find it confusing to understand what is going on.